### PR TITLE
privacy.anonymization added with test

### PR DESCRIPTION
**Pull request by aminatkhamokova**
privacy folder with mitigation (anonymization) added with the corresponding test

### CODE REVIEW

File: holisticai/privacy/mitigation/_Anonymization.py [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
```
Comment (981526092) [inline holisticai/privacy/mitigation/_Anonymization.py]: @aminatkhamokova Need to change the citation similar to below format with comment:

Adversarial Robustness Toolbox Citation
Nicolae, M.-I., Sinn, M., Tran, M.-N., Buesser, B., Rawat, A., Wistuba, M., Zantedeschi, V., Baracaldo, N., Chen, B., Ludwig, H., Molloy, I., & Edwards, B. (2018). Adversarial Robustness Toolbox v1.2.0. CoRR, 1807.01069.

Retrieved from https://arxiv.org/pdf/1807.01069

GitHub Repository: https://github.com/Trusted-AI/adversarial-robustness-toolbox

**Review by crismunoz — COMMENTED** [review]
Please add some tutorials

File: tests/privacy/mitigation/test_anonymization.py
Comment (crismunoz) [inline tests/privacy/mitigation/test_anonymization.py:1]: Does the strategy works for classification and regression? In that case you must create a test for each problem type.

**Review by crismunoz — COMMENTED** [review]
This anonymiztion works as preprocessing? Should it be include as a stage in the sklearn pipeline during training?

**Comment by 981526092** [comment]
> Please add some tutorials

Hello Cristian,

We have created tutorials and plans to push them sequentially.

File: tests/privacy/mitigation/test_anonymization.py
Comment (981526092) [reply tests/privacy/mitigation/test_anonymization.py:1]: > Does the strategy works for classification and regression? In that case you must create a test for each problem type.

We plan to first work on the classification problem. Then expand to regression. Then the rests.

**Review by 981526092 — COMMENTED** [review]
Valid Reference

**Review by 981526092 — COMMENTED** [review]
Valid Reference

**Review by 981526092 — APPROVED** [review]
Valid PR

**Pull request merged**